constants.xml

```diff
@@ -145,6 +145,11 @@
         <!-- Shard size will be automatically calculated if COMM_SIZE = 0 -->
         <COMM_SIZE>200</COMM_SIZE>
         <NUM_DS_ELECTION>5</NUM_DS_ELECTION>
+        <!-- DS Node must meet this minimum threshold to remain in the committee -->
+        <DS_PERFORMANCE_THRESHOLD_PERCENT>0.05</DS_PERFORMANCE_THRESHOLD_PERCENT>
```

Comment:
DS_PERFORMANCE_THRESHOLD_PERCENT and NUM_DS_BYZANTINE_REMOVED looks never used in the code

Reply:
@ShengguangXiao DSBlockPreProcessing.cpp

Reply:
Used here: https://github.com/Zilliqa/Zilliqa/pull/1587/files#diff-76372bc310da6631dec75919a39cb4e2R1426